milestone2.ts

```diff
@@ -92,23 +98,32 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data) => {
+  let index = data.findIndex(x => x.city === "SF");
+  return index
+};
 
 //console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the volunteers from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data, n) => {
+  let d2 = data.filter(x => x.age > n)
+  return d2.filter(x => x.state === "CA");
```

Comment:
This can be reduced to one filter using the logical and (&&)
```suggestion
  let d2 = data.filter(x => x.age > n && x.state === "CA");
```